playbooks/adhoc/sdn_restart/oo-sdn-restart.yml

```diff
@@ -49,5 +49,6 @@
       name: "{{ item }}"
       state: restarted
     with_items:
-    - oso-f22-host-monitoring
-    - oso-rhel7-zagg-client
+#    - oso-f22-host-monitoring
```

Comment:
No need to leave these commented out. Just remove them entirely, and leave only oso-rhel7-host-monitoring.